File: Unilink/Flutter/lib/models/university.dart
class University {
  String id;
  String name;
  String sortName;
  bool status;
  University({this.id, this.name, this.sortName, this.status});
  factory University.jsonFrom(Map<String, dynamic> json) {
    return University(
        id: json["id"],
        name: json["name"],
        sortName: json["sortName"],
        status: json["status"] == "true");
  }
}

File: Unilink/DotNet/Unilink.Data/Entity/University.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Unilink.Data.Entity
{
    [Table("University")]
    public class University
    {
        [Key]
        public Guid Id { get; set; }
        [Required]
        public string Name { get; set; }
        [Required]
        public string SortName { get; set; }
        [Required]
        public bool Status { get; set; }
    }
}
